lib/static/navigation.dart

```diff
@@ -26,9 +27,9 @@ Future<void> connect() async {
   final configString = await rootBundle.loadString('cptclient.yaml');
   final dynamic configMap = loadYaml(configString);
 
-  html.window.localStorage.putIfAbsent('ServerScheme', () => configMap['ServerScheme']);
-  html.window.localStorage.putIfAbsent('ServerHost', () => configMap['ServerHost']);
-  html.window.localStorage.putIfAbsent('ServerPort', () => configMap['ServerPort']);
+  html.window.localStorage.putIfAbsent('ServerScheme', () => Env.serverScheme.fromString() ?? configMap['ServerScheme']);
+  html.window.localStorage.putIfAbsent('ServerHost', () => Env.serverHost.fromString() ?? configMap['ServerHost']);
+  html.window.localStorage.putIfAbsent('ServerPort', () => Env.serverPort.fromInt()?.toString() ?? configMap['ServerPort']);
```

Comment:
`dart-define` take precedence over `cptclient.yaml` as the file is required. But one doesn't need to specify them.